.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7.2
+2.7.2
```

Comment:
While rbenv handles this fine when setting the environment, it can't use it to trigger install with `rbenv install`. This change simplifies the setup process. This lets us tell the user to use `rbenv install` rathern than needing to get them to `cat .ruby-version` or to keep a version number up to date in the documentation.